Carbonplus-Grasslands/cavan-compact.json

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
What about renaming this folder to `CarbonPlus-Grasslands`?

Reply:
sgtm